qiskit_ibm_provider/utils/backend_converter.py

```diff
@@ -132,9 +132,10 @@ def convert_to_target(
                             target[inst][(qubit,)].calibration = sched
                     else:
                         target[inst][qarg].calibration = sched
-    target.add_instruction(
-        Delay(Parameter("t")), {(bit,): None for bit in range(target.num_qubits)}
-    )
+    if "delay" not in target.operation_names:
```

Comment:
You can just do:

```suggestion
    if "delay" not in target:
```

and avoid the intermediate object creation.